modules/apps/transfer/types/msgs_test.go

```diff
@@ -38,6 +40,9 @@ var (
 	zeroCoin         = sdk.Coin{Denom: "atoms", Amount: sdk.NewInt(0)}
 
 	timeoutHeight = clienttypes.NewHeight(0, 10)
+
+	validAuthority = sdk.AccAddress("authority").String()
```

Comment:
Sorry, maybe I'm missing something but how can this be a valid authority address?

Reply:
It's not valid, that variable is unused and a mistake, thanks for catching, instead, I redefined it down the line

Reply:
Yeah, doesn't make much sense. Test passes only because it is wrapped by `AccAddress`. I took this test case from SDK I believe, I can remove it if you'd like.